<?xml version="1.0"?>
<launch>
  <rosparam command="load" file="$(find path_tracking)/params/path_tracking_sim.yaml"/>
  
  <node pkg="control_si" type="control_si" name="control_si" output="screen"/>

  <node 
    pkg="rosservice" 
    type="rosservice" 
    name="set_path_tracking_log_level" 
    args="call --wait /path_tracking/set_logger_level 'ros.path_tracking' 'info'" />

  <!-- <node pkg="ptp_trajectory" type="ptp_trajectory" name="ptp_trajectory" output="screen"/> -->
  
  <node pkg="path_tracking" type="path_tracking" name="path_tracking" output="screen"/>
  
</launch>
